src/styles/styles.css

```diff
@@ -9,7 +9,51 @@
     box-sizing: border-box;
 }
 
+#container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+/* ********  START PAGE  ******** */
+
+#le {
+    font-family: 'Black Ops One', cursive;
+    font-size: 4rem;
+}
+
+#instructions {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.instructions-header {
+    font-family: 'Merriweather', serif;
+    padding-bottom: 1rem;
+}
+
+.instructions-paragraph {
+    width: 60%;
+    border: 3px solid #f3f2c9;
+    padding: 4rem;
+    line-height: 1.5;
+    text-align: center;
+}
+
+/* ********  GAME-PAGE  ******** */
+
+#game-page {
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    top: 0;
+    z-index: 0;
+}
+
 #game-board {
+    background-image: url('../Game_board1.jpg');
```

Comment:
background image is repeating. 
![image](https://user-images.githubusercontent.com/62726177/127319498-4897a7df-ec52-426d-82fe-e3026048ebe3.png)

Reply:
I think we should remove the image for now and do that when we implement the whole design.

Reply:
fixed boss